clayui.com/content/docs/components/label.mdx

```diff
@@ -22,6 +22,12 @@ import {
 </div>
 </div>
 
+## Overview
+
+`ClayLabel` offers two different APIs for use by toggling the prop `withClose`. By default(`withClose={true}`), `ClayLabel` behaves like a high-level component. If you want to use the lower-level components and compose multiple parts to your label, all you need to do is set `withClose={false}`.
+
+Check out [this storybook example](storybook.clayui.com/?path=/story/components-claylabel--w-content-before) for a demo.
+
```

Comment:
@bryceosterhaus as it is a typical feature that we don't want to support for a long time, it might be worth adding a warning about it and that we will remove this double behavior in a major version later. What do you think?

Reply:
@matuzalemsteles I'm thinking its probably best to not mention that since they are essentially toggling the intended behavior of what this component will be. The next major version will likely behave the same even if `withClose={false}` is set.

When we do ever release 4.x, we can then note that `with*` props are moved to high-level components. But overall, I think it'd be unnecessary to mention at the moment since 4.x isn't really even on the radar yet. Once we get 4.x on the radar and have a better idea of what it would be, we can always add the note then.

Reply:
Yeah, no problem. LGTM